cli/js/os.ts

```diff
@@ -140,6 +140,162 @@ export function homeDir(): string {
   return path;
 }
 
+/**
+ * Returns the current user's cache directory.
+ * Requires the `--allow-env` flag.
+ */
+export function cacheDir(): string {
+  const path = sendSync(dispatch.OP_CACHE_DIR);
```

Comment:
Maybe combine all these into a single op?
``` TypeScript
const path = sendSync(dispatch.OP_GET_DIR, { type: "cache" })
```

Reply:
done! but `type` is a rust keyword. so I use `{ name: "cache" }` instead